substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/ResourceStorageEntry.java

```diff
@@ -0,0 +1,31 @@
+package com.oracle.svm.core.jdk;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ResourceStorageEntry {
```

Comment:
All of these should go into a separate package.

Reply:
Done.